### ISSUE TITLE
I use your tps-resnet-bilstm-attn code to train my handwriten data，the acc is 0.729;if I use the tps-resnet-bilstm-attn code of deep-text-recognition-benchmark,the acc is 0.805.Meanwhile,the acc of small_satrn is only 0.09

### ISSUE THREAD

**Issue by SeanLiu081** [Media-Smart/vedastr#43]

**Comment by SeanLiu081** [comment]
![image](https://user-images.githubusercontent.com/26735847/97651947-4fbbf000-1a98-11eb-8925-83568fc7caaf.png)

**Comment by SeanLiu081** [comment]
![image](https://user-images.githubusercontent.com/26735847/97652127-a88b8880-1a98-11eb-92d8-bdd94d0a6b72.png)
the result of tps-resnet-blistm-attn

**Comment by ChaseMonsterAway** [comment]
I think you should check the hyperparameters first. For example, i think you should change the lr-scheduler of small-satrn. Besides, can you paste the training log of tps-resnet-bilstm-attn of deep-text-recognition-benchmark?

**Comment by SeanLiu081** [comment]
![image](https://user-images.githubusercontent.com/26735847/97665454-62ddb880-1ab6-11eb-9c0a-07a29f8bc01e.png)
 The training log of tps-resnet-bilstm-attn of deep-text-recognition-benchmark,If the number of iterations is the same (100000), the acc of deep-text-recognition-benchmark is 0.795 while the acc of vedastr is only 0.73

**Comment by ChaseMonsterAway** [comment]
This accuracy of deep-text-recognition-benchmark is the validation accuracy. The accuracy in the log of vedastr with prefix "Train, Epoch" is the train infomations. So you should check the validation accuracy by searching 'validating' in training log.

**Comment by ChaseMonsterAway** [comment]
The code of deep-text-recognition-benchmark:

![image](https://user-images.githubusercontent.com/58807745/97669693-a7218680-1abf-11eb-93d0-27c191b9d5c9.png)

**Comment by ChaseMonsterAway** [comment]
Is it possible for you to show me the whole training log by Vedastr? Can you upload it?

**Comment by SeanLiu081** [comment]
I did not add the validation data during training, but I tested it on test data with the latest model. The accuracy is 0.709,the picture is the result of the test, and I uploaded the training log.
![image](https://user-images.githubusercontent.com/26735847/97676329-721b3100-1acb-11eb-9675-f2d07e81b42d.png)


[20201029_183221.log](https://github.com/Media-Smart/vedastr/files/5463774/20201029_183221.log)

**Comment by ChaseMonsterAway** [comment]
I'm not sure what's the problem now. Can you add the group chat?
![gg](https://user-images.githubusercontent.com/58807745/97680669-07b9bf00-1ad2-11eb-90ac-fec9da91b917.png)

**Comment by ChaseMonsterAway** [comment]
We have talk about this problem.and the reasons are summarized as follows:
 1. `shuffle=True`. If you don't use balace sampler or other defined sampler, you should add shuffle=True in train dataloader. This is same with the behavior of dataloader in pytorch. 
 2. Different data filtering condition. If you use the code of deep-text-recognition-benchmark, you should carefully check the filtering condition when test. For more infomation, you can check the filter condition in [vedastr/datasets/base.py](https://github.com/Media-Smart/vedastr/blob/master/vedastr/datasets/base.py) or [deep-text-recognition-benchmark/blob/master/dataset.py](https://github.com/clovaai/deep-text-recognition-benchmark/blob/master/dataset.py).
3. After solved these two problems, the accuracy of vedastr is 75.16 while deep-text-recognition-benchmark is 74.58.
